src/components/Column.vue

```diff
@@ -76,6 +87,21 @@ export default defineComponent({
         task: {},
       });
     },
+    isSectionComplete(columnName: string) {
+      return columnName === 'Done' || columnName.startsWith('Complete') || columnName.startsWith('Finish')
+    },
+    release(sectionId: string) {
```

Comment:
This feels like it should be a mutation or action as I can see it being called from elsewhere